lib/internal/fetch/headers.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
+
+const {
+  codes: { ERR_INVALID_ARG_VALUE },
+} = require('internal/errors');
+
+const {
+  ObjectEntries,
+  ArrayIsArray,
+  StringPrototypeToLocaleLowerCase,
+  StringPrototypeReplace,
+  Symbol,
+  SymbolIterator,
+  Array,
```

Comment:
```suggestion
  Array,
  ArrayIsArray,
  MathFloor,
  ObjectEntries,
  StringPrototypeReplace,
  StringPrototypeToLocaleLowerCase,
  Symbol,
  SymbolIterator,
```
lexicographical order + `MathFloor`